compat/src/index.js

```diff
@@ -34,6 +34,17 @@ import {
 	__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED
 } from './render';
 
+const requestIdleCallbackPreact =
+	(typeof self !== "undefined" &&
+		self.requestIdleCallback &&
+		self.requestIdleCallback.bind(window)) ||
+	function (cb) {
+		return queueMicrotask(() => {
+			cb();
+		});
+	};
+
```

Comment:
I'm not sure if we need to bind the this value explicitly. Maybe we can simplify this a bit?

```suggestion
const requestIdleCallbackPreact =
	typeof requestIdleCallback === 'function'
		? requestIdleCallback
		: queueMicrotask;
```

Reply:
Hi @marvinhagemeister , Currently i updated to `setTimeout` since it need to be cleared in unmount. (Can you please help me with it)